src/braket/aws/aws_session.py

```diff
@@ -157,25 +160,20 @@ def _update_user_agent(self) -> None:
         """
         Updates the `User-Agent` header forwarded by boto3 to include the braket-sdk,
         braket-schemas and the notebook instance version. The header is a string of space delimited
-        values (For example: "Boto3/1.14.43 Python/3.7.9 Botocore/1.17.44"). See:
-        https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html#botocore.config.Config
```

Comment:
nit: is this link no longer relevant?

Reply:
The old code reached into the internal botocore config object sitting inside of the client, `braket_client._client_config`. The new path does not use the config object.